src/components/contextual/pages/vebal/LMVoting/LMVoting.vue

```diff
@@ -277,3 +277,9 @@ function handleVoteSuccess() {
     />
   </teleport>
 </template>
+
+<style scoped>
+.filter-input {
+  flex: 1 1 100%;
```

Comment:
Again, can this not be achieved with tailwind classes?

Reply:
it can, but I checked one more time and it seems that there is no need in this